src/main/java/cpsc/dlsproject/tools/VariableMap.java

```diff
@@ -0,0 +1,13 @@
+package cpsc.dlsproject.tools;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class VariableMap {
+
+    public static final Map<String, Variable> variables = new HashMap<>();
+
+    public static Map<String, Variable> getVariableMap(){
+        return variables;
+    }
+}
```

Comment:
Delete this as discussed verbally yesterday. This is not used anywhere in the code.